gfreshii-webapp/src/main/java/com/edbootcamp/view/RecipeImpl.java

```diff
@@ -15,6 +17,7 @@
 	@JsonProperty("instruction")
 	private String instruction;	
 	@JsonProperty("ingredients")
+	@JsonIgnoreProperties(value= {"ingredients"})
 	private List<IngredientImpl> ingredients;
```

Comment:
Ok your Jackson problem here isn't that you're trying to put in a collection of ingredients. But that they are "Impl" types and not "View" types.  Impl types are likely to have recursive relational references (ex. Between ingredents and recipe, that don't convert).   Use a "View" object without a recursive reference, and you can return a list of ingredients without using JsonIgnore